Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -978,6 +990,50 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Overload Swift's 'contains' operator for matching regex
+    ///
+    /// - Parameter lhs: String to check on regex.
+    /// - Parameter rhs: Regex to match against.
+    /// - Returns: true if there is at least one match for the regex in the string.
+    static func ~= (lhs: String, rhs: NSRegularExpression) -> Bool {
+        let range = NSRange(location: 0, length: lhs.utf16.count)
+        return rhs.firstMatch(in: lhs, options: [], range: range) != nil
```

Comment:
```suggestion
        return rhs.firstMatch(in: lhs, range: range) != nil
```
You can omit the default value.